### PR TITLE
chore!: drop node 6 support

### PR DESCRIPTION
**Pull request by DominicKramer**
BREAKING CHANGE: Node 6 is no longer tested.

### CODE REVIEW

**Comment by DominicKramer** [comment]
This shouldn't land until PR #676 lands.

**Comment by DominicKramer** [comment]
PR #676 has landed, and thus this PR is no longer blocked.

**Review by jkwlui — APPROVED** [review]

**Comment by JustinBeckwith** [comment]
This was covered in #680 and #679

**Pull request closed without merging**
